### PR TITLE
Add no-zip Parameter Flag to Documenation

### PR DESCRIPTION
**Pull request by r0b2g1t**
This PR add the no-zip feature to the documentation with an description.

### CODE REVIEW

File: pcweb/pages/docs/hosting/self_hosting.py [hunk 1]
```diff
@@ -93,6 +93,14 @@ def self_hosting():
             pc.code("--backend-only"),
             " flags.",
         ),
+        doctext(
+            "It is also possible to export the components without zip it separate. To do this, use the ",
+            pc.code("--no-zip"),
+            " parameter.",
```
Comment (picklelo) [inline pcweb/pages/docs/hosting/self_hosting.py:99]: Can we add a space after the `.` here. We need a way to catch this automatically as it's easy to miss.

File: pcweb/pages/docs/hosting/self_hosting.py [hunk 2]
```diff
@@ -93,6 +93,14 @@ def self_hosting():
...
+            "This provides the frontend in the ",
+            pc.code("_static/"),
```
Comment (picklelo) [inline pcweb/pages/docs/hosting/self_hosting.py:101]: Should this be the `.web/_static/` directory?

**Review by picklelo — COMMENTED** [review]
Thanks for adding docs! Super helpful - just small comments

File: pcweb/pages/docs/hosting/self_hosting.py (hunk 1 above)
Comment (r0b2g1t) [reply pcweb/pages/docs/hosting/self_hosting.py:99]: I have add the space

File: pcweb/pages/docs/hosting/self_hosting.py (hunk 2 above)
Comment (r0b2g1t) [reply pcweb/pages/docs/hosting/self_hosting.py]: I have add .web/ in front of _static/

**Review by picklelo — APPROVED** [review]
nice!

**Pull request merged**